generators/app/constants.js

```diff
@@ -92,6 +92,8 @@ module.exports.TEMPLATE_FILES = [
   `${COMPONENTS_PATH}/TextArea/index.js`,
   `${COMPONENTS_PATH}/Checkbox/index.js`,
   `${COMPONENTS_PATH}/Checkbox/layout.js`,
+  `${COMPONENTS_PATH}/RadioGroup/index.js`,
+  `${COMPONENTS_PATH}/RadioGroup/components/RadioOption/index.js`,
```

Comment:
It would add `README.md` and `docs/RadioGroup.md`

Reply:
True but you are already adding them in your Login pr.
I think i will let this as is to avoid conflicts.
Let me know.